manageiq-api-client.gemspec

```diff
@@ -6,6 +6,7 @@ require 'manageiq/api/client/version'
 Gem::Specification.new do |spec|
   spec.name          = "manageiq-api-client"
   spec.version       = ManageIQ::API::Client::VERSION
+  spec.required_ruby_version = '>= 3.0'
```

Comment:
This is a weird spot within the file to put this.  I think typically this line comes before the dependencies list in it's own paragraph, so let's move it down this.

Reply:
haha, I was following the convention we had in a few where it followed the version, like above.  I agree.  I can move it.

Reply:
done

Reply:
oh weird - we should change those other ones then, too